js/packages/phoenix-client/src/schemas/llm/phoenixPrompt/responseFormatSchema.ts

```diff
@@ -1,18 +1,24 @@
 import z from "zod";
-import { jsonSchemaZodSchema } from "../../jsonSchema";
+import { schemaMatches } from "../../../utils/schemaMatches";
+import { PromptResponseFormat } from "../../../types/prompts";
```

Comment:
probably need to shuffle around this directory since it's not clear that this type relates to the REST API based types (e.g. openapi)